vllm/engine/arg_utils.py

```diff
@@ -451,6 +453,16 @@ class EngineArgs:
     kv_sharing_fast_prefill: bool = \
         CacheConfig.kv_sharing_fast_prefill
 
+    verification_method: Optional[str] = None
+    signature: Optional[str] = None
+    ignore_git_paths: bool = True
+    identity: str = ""
+    identity_provider: str = ""
+    use_staging: bool = False
+    certificate_chain: Optional[Iterable[str]] = None
+    log_fingerprints: bool = False
+    public_key: Optional[str] = None
```

Comment:
Please get the defaults from `SignatureVerificationConfig` so that they will always be synchronised. e.g.:

```
    verification_method: Optional[str] = SignatureVerificationConfig.verification_method
    ...
```